buildconfig/CMake/LinuxPackageScripts.cmake

```diff
@@ -162,49 +162,30 @@ elif [ -n \"\${TLSESSIONDATA}\" ]; then  # running in thin-linc
   fi
 fi" )
 
-# The scripts need tcmalloc to be resolved to the runtime library as the plain
+# The scripts need jemalloc to be resolved to the runtime library as the plain
 # .so symlink is only present when a -dev/-devel package is present
-if ( TCMALLOC_FOUND )
-  get_filename_component ( TCMALLOC_RUNTIME_LIB ${TCMALLOC_LIBRARIES} REALPATH )
+if ( JEMALLOC_FOUND )
```

Comment:
Just one more correction after updating the finder name. On a clean configure it now skips this and the jemalloc line is not filled in within the scripts.